GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg

```diff
@@ -363,463 +581,235 @@ PART
-	{
-		name = TacGenericConverter
-		converterName = CO2 LiOH Scrubber
-		tag = Life Support
-		StartActionName = Start CO2 Scrubber
-		StopActionName = Stop CO2 Scrubber
-		conversionRate = 2.0
-		GeneratesHeat = False
-		UseSpecialistBonus = False
-
-		INPUT_RESOURCE
-		{
-			ResourceName = ElectricCharge
-			Ratio = 0.1
-		}
+    @MODULE[ModuleFuelTanks]
+    {
+        @volume += 165
+
+        TANK
+        {
+            name = Food
+            amount = 23.4
+            maxAmount = 35.1
+        }
+
+        //  Two water tanks with a capacity of ~20 kg each.
+
+        TANK
+        {
+            name = Water
+            amount = 40
+            maxAmount = 40
+        }
+
+        //  Two oxygen tanks with a capacity of ~1.1 kg each.
+
+        TANK
+        {
+            name = Oxygen
+            amount = 1563.4
+            maxAmount = 1563.4
+        }
+
+        TANK
+        {
+            name = LithiumHydroxide
+            amount = 4.1
+            maxAmount = 6.2
+        }
+
+        TANK
+        {
+            name = Waste
+            amount = 0
+            maxAmount = 3.2
+        }
+
+        TANK
+        {
+            name = WasteWater
+            amount = 0
+            maxAmount = 29.3
+        }
+
+        TANK
+        {
+            name = CarbonDioxide
+            amount = 0
+            maxAmount = 1023
+        }
+    }
 
-		INPUT_RESOURCE
-		{
-			ResourceName = CarbonDioxide
-			Ratio = 0.00589121
-		}
+    !MODULE[TacGenericConverter],*{}
 
-		INPUT_RESOURCE
-		{
-			ResourceName = LithiumHydroxide
-			Ratio = 0.00001189
-		}
-
-		OUTPUT_RESOURCE
-		{
-			ResourceName = Waste
-			Ratio = 0.00003847
-			DumpExcess = False
-		}
-	}
+    MODULE
+    {
+        name = TacGenericConverter
+        converterName = CO2 LiOH Scrubber
+        tag = Life Support
+        StartActionName = Start CO2 Scrubber
+        StopActionName = Stop CO2 Scrubber
+        conversionRate = 2.0
+        GeneratesHeat = False
+        UseSpecialistBonus = False
+
+        INPUT_RESOURCE
+        {
+            ResourceName = ElectricCharge
+            Ratio = 0.1
+        }
+
+        INPUT_RESOURCE
+        {
+            ResourceName = CarbonDioxide
+            Ratio = 0.00589121
+        }
+
+        INPUT_RESOURCE
+        {
+            ResourceName = LithiumHydroxide
+            Ratio = 0.00001189
+        }
+
+        OUTPUT_RESOURCE
+        {
+            ResourceName = Waste
+            Ratio = 0.00003847
+            DumpExcess = False
+        }
+    }
 }
 
-//	================================================================================
-//	Add RasterPropMonitorComputer Module
-//	================================================================================
+//  ================================================================================
+//	Configure RemoteTech
+//  ================================================================================
 
-@PART[ROC-LEMAscent]:NEEDS[RasterPropMonitor]
```

Comment:
Why did you break this patch structure, and not include NEEDS[RasterPropMonitor] when you declared its fields above instead?